libs/cde-visualization/src/lib/cde-visualization/cde-visualization.component.html

```diff
@@ -2,6 +2,7 @@
   <cde-visualization-header
     (downloadNodes)="downloadNodes()"
     (downloadEdges)="downloadEdges()"
+    homeLink="home"
```

Comment:
The cde-visualization is supposed to take this as an input, not hardcoded in it's html...